eth/1-security-council-phase-0/README.md

```diff
@@ -0,0 +1,202 @@
+multisig signers to verify the correctness of the onchain operations.
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
+just install
+cd eth/1-security-council-phase-0
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message "Application
+is ready".
+
+### 3. Simulate and validate the transaction
+
+Make sure your ledger is still unlocked and run the following.
+
+Remember that by default just is running with the address derived from
+`/0` (first nonce). If you wish to use a different account, run `just
+simulate [X]`, where X is the derivation path of the address
+that you want to use.
+
+``` shell
+just simulate
+```
+
+You will see a "Simulation link" from the output.
+
+Paste this URL in your browser. A prompt may ask you to choose a
+project, any project will do. You can create one if necessary.
+
+Click "Simulate Transaction".
+
+We will be performing 3 validations and extract the domain hash and
+message hash to approve on your Ledger:
+
+1. Validate integrity of the simulation.
+2. Validate correctness of the state diff.
+3. Validate and extract domain hash and message hash to approve.
+
+#### 3.1. Validate integrity of the simulation.
+
+Make sure you are on the "Overview" tab of the tenderly simulation, to
+validate integrity of the simulation, we need to check the following:
+
+1. "Network": Check the network is Ethereum Mainnet.
+2. "Timestamp": Check the simulation is performed on a block with a
+   recent timestamp (i.e. close to when you run the script).
+3. "Sender": Check the address shown is your signer account. If not,
+   you will need to determine which “number” it is in the list of
+   addresses on your ledger. By default the script will assume the
+   derivation path is m/44'/60'/0'/0/0. By calling the script with
+   `just simulate 1` it will derive the address using
+   m/44'/60'/1'/0/0 instead.
+
+![](./images/tenderly-overview-network.png)
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+1. There is only a single state override at address
+   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`, which overrides
+   storage slot `0x4` to new value `0x1`. This override is only
+   intended to change the Foundation multisig's quorum threshold to 1
+   so we can perform a tenderly simulation of the execution,
+2. The `ProxyAdmin` contract at
+   `0x543ba4aadbab8f9025686bd03993043599c6fb04`'s `_owner` is changed
+   to a new multisig, and the configuration of this new multisig
+   correctly implements the two approved proposals:
+   1. [Security Council: Vote #1](https://vote.optimism.io/proposals/27439950952007920118525230291344523079212068327713298769307857575418374325849).
+   2. [Security Council Membership Ratification](https://link.to/be/determined).
```

Comment:
I don't see the `0xdc53b1d440b8a56dd50a6d69312dac903d3aa48b` address listed in the proposal. 

How does that flow of information work?